section3/reflection.md

```diff
@@ -1,17 +1,27 @@
 ## Section 3 Reflection
 
 1. How different did your workflow feel this week, considering we asked you to follow the Pomodoro technique?
+- Sometimes I feel the technique is very useful but other times I do feel as though it risks interrupting my work flow. I like the idea of more bite sized pieces of work.
```

Comment:
Understandable!! Just make sure you give yourself a break sometime - different folks like different lengths of a work chunk, so do what works for you.